posts/2021-07-08-logging-format-21008-beta.adoc

```diff
@@ -0,0 +1,274 @@
+---
+layout: post
+title: "MicroProfile 4.1 plus more exciting features now in Open Liberty 21.0.0.8-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: MicroProfile 4.1 plus more exciting features now in Open Liberty 21.0.0.8-beta - OpenLiberty.io
+seo-description: Open Liberty 21.0.0.8-beta introduces MicroProfile 4.1 with updates to the MicroProfile Health feature. A new logging format (`TBASIC`) is made available. The Stale Connection Identification feature makes it possible to provide additional configuration for a data source. A couple of Jakarta EE 9.1 convenience features are also included in the "All Beta" package.
+blog_description: Open Liberty 21.0.0.8-beta introduces MicroProfile 4.1 with updates to the MicroProfile Health feature. A new logging format (`TBASIC`) is made available. The Stale Connection Identification feature makes it possible to provide additional configuration for a data source. A couple of Jakarta EE 9.1 convenience features are also included in the "All Beta" package.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile 4.1 plus more exciting features now in Open Liberty 21.0.0.8-beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+Open Liberty 21.0.0.8-beta introduces MicroProfile 4.1 with updates to the MicroProfile Health feature. A new logging format (`TBASIC`) is made available. The Stale Connection Identification feature makes it possible to provide additional configuration for a data source. A couple of Jakarta EE 9.1 convenience features are also included in the <<allbeta, "All Beta" package>>.
+
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try one of these packages, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
+
+* <<mp, MicroProfile Updates>>
+** <<mp4.1, MicroProfile 4.1>>
+** <<mpHealth3.1, MicroProfile Health 3.1>>
+* <<tbasic, TBASIC Logging Format Beta>>
+* <<staleConnectionID, Customizing Stale Connection Identification>>
+* <<JEE9.1, Jakarta EE 9.1 convenience features>>
+
```

Comment:
In the preview in github it shows that the mp4.1, mpHealth3.1 and JEE9.1 links are not working.  I am guessing because of the period in the name.  Is it working on the draft server?  May need to change to use underscores?

Reply:
Ahh, thank you! I haven't noticed that.